Command/DoctrineCommand.php

```diff
@@ -1,66 +1,106 @@
+                    $error['message']
+                ));
             }
+
             $configuration->setMigrationsDirectory($dir);
         } else {
-            $dir = $configuration->getMigrationsDirectory();
             // class Kernel has method getKernelParameters with some of the important path parameters
-            $pathPlaceholderArray = array('kernel.root_dir', 'kernel.cache_dir', 'kernel.logs_dir');
+            $pathPlaceholderArray = ['kernel.root_dir', 'kernel.cache_dir', 'kernel.logs_dir'];
+
             foreach ($pathPlaceholderArray as $pathPlaceholder) {
-                if ($container->hasParameter($pathPlaceholder) && preg_match('/\%'.$pathPlaceholder.'\%/', $dir)) {
-                    $dir = str_replace('%'.$pathPlaceholder.'%', $container->getParameter($pathPlaceholder), $dir);
+                if (! $container->hasParameter($pathPlaceholder) || ! preg_match('/\%' . $pathPlaceholder . '\%/', $dir)) {
+                    continue;
                 }
+
+                $dir = str_replace('%' . $pathPlaceholder . '%', $container->getParameter($pathPlaceholder), $dir);
             }
-            if (!is_dir($dir) && !@mkdir($dir, 0777, true) && !is_dir($dir)) {
+
+            if (! is_dir($dir) && ! @mkdir($dir, 0777, true) && ! is_dir($dir)) {
                 $error = error_get_last();
-                throw new \ErrorException($error['message']);
+
+                throw new ErrorException(sprintf(
+                    'Failed to create directory "%s" with message "%s"',
+                    $dir,
+                    $error['message']
+                ));
             }
+
             $configuration->setMigrationsDirectory($dir);
         }
-        if (!$configuration->getMigrationsNamespace()) {
+
+        if (empty($configuration->getMigrationsNamespace())) {
             $configuration->setMigrationsNamespace($container->getParameter('doctrine_migrations.namespace'));
         }
-        if (!$configuration->getName()) {
+
+        if (empty($configuration->getName())) {
             $configuration->setName($container->getParameter('doctrine_migrations.name'));
         }
+
         // For backward compatibility, need use a table from parameters for overwrite the default configuration
-        if (!($configuration instanceof AbstractFileConfiguration) || !$configuration->getMigrationsTableName()) {
+        if (! ($configuration instanceof AbstractFileConfiguration) || empty($configuration->getMigrationsTableName())) {
             $configuration->setMigrationsTableName($container->getParameter('doctrine_migrations.table_name'));
         }
+
+        $configuration->setMigrationsColumnName($container->getParameter('doctrine_migrations.column_name'));
+        $configuration->setMigrationsColumnLength($container->getParameter('doctrine_migrations.column_length'));
+        $configuration->setMigrationsExecutedAtColumnName($container->getParameter('doctrine_migrations.executed_at_column_name'));
```

Comment:
I am not sure I understand what you mean. You mean how the others have an `if (empty(...))` condition around them? These values in the Configuration class have a default value so it can't work the same way unless you have another thought/idea?